python/python/glide/async_commands/core.py

```diff
@@ -2877,6 +2878,57 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def bzmpop(
+        self,
+        keys: List[str],
+        modifier: ScoreFilter,
+        timeout: float,
+        count: Optional[int] = None,
+    ) -> Optional[List[Union[str, Dict[str, float]]]]:
```

Comment:
I wonder if we should create a special object for this in redis-rs, since Java returns a list of object and it doesn't looks nice

Reply:
I'm not sure I understand. Are you suggesting we change the returned data format in value_conversion.rs? Is there an existing example of your suggestion that I can take a look at? One thing we could do to make this a bit clearer is create a Python type alias (see [here](https://docs.python.org/3/library/typing.html#type-aliases)) and put it in a new file called types.py?

Reply:
I am working on [GeoSearch](https://redis.io/docs/latest/commands/geosearch/) command (and also [LCS](https://redis.io/docs/latest/commands/lcs/), it has the same complexity), if you take a look at it, it has a super complicated return type, so I thought about adding a type to Redis Value in redis-rs named geosearch or something, and for this command I will simply return List[GeoSearch], (same udea as we imported script - if you ask how can we do that convertion from rust to any language)

I guess here its not so necessary, since the return type is consistent, but ig we will discuss that later

Reply:
@shohamazon 
We can create a simple object with (nullable) optional entries (say: GeoLocation) and a constructor that takes an Object[].  This would work as long as one of WITHDIST, WITHCOORD, WITHHASH is included. 

```
4) 1) "edge1"
   2) "279.7405"
   3) 1) "12.7584877610206604"
      2) "38.78813451624225195"
```

cc: @tjzhang-BQ